definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4525,7 +4532,7 @@
     "LiveFrom": "01/03/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorApplyForDN",
-    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
are we adding permissions to PETSOL, or we replace caseworker-divorce-solicitor with PETSOL?

Is this correct that all other caseworker-divorce-solicitor will not be able to do it anymore?